src/App.js

```diff
@@ -1,26 +1,55 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+// import Header from './Header.js';
+import HomePage from './HomePage.js'
+import DetailPage from './DetailPage'
+import List from './List'
+import ListItem from './ListItem'
+import AboutMe from './AboutMe'
+import {
+    BrowserRouter as Router, 
+    Route, 
+    Switch,
+} from 'react-router-dom';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
 
-export default App;
+
+  export default class App extends Component {
+    render() {
+        return (
+            <>
+            <div>
+                <Router>
+                    <Switch>
+                        <Route 
+                            path="/butterfly" 
```

Comment:
this should just be '/'  so the router knows what to display when you first load the page.

Reply:
Thank you! Just resubmitted in canvas and updated the dev branch, so this should be fixed and upon load, my list objects are showing up.  Hopefully that fixed it. Thanks Sarah!